### DISCUSSION TITLE
Github Copilot Icon in Error List

### DISCUSSION THREAD

**Discussion by Trinity-Kerr** [community/community#126634] [Copilot Conversations] [answered]
### Select Topic Area

Question

### Body

I went to the build day for MS and saw an awesome presentation about copilot.  I want to demo one of the features, but even after updating my visual studio to 17.10 I am unable to see the github copilot icon in my error list.  Is there a setting I have to change for this to show up and be used? 

This is a screen clip from the demo I watched, my environment does not show the icon:
![Untitledpicture](https://github.com/community/community/assets/135732091/17b36722-4d97-4720-802b-5908288d1ba6)

I am also not seeing the "explain failure with copilot" in my Test Explorer.  They said this was already available in 17.10  as well.

**Comment by ayim** [comment, marked as answer]
Hi @Trinity-Kerr from Visual Studio team

1) The error list feature isn't available to the public yet, I recommend following the DevBlogs or our LinkedIn page to stay in touch for its launch
https://devblogs.microsoft.com/visualstudio/

2) The explain test failure with Copilot feature should be available.
If you have Visual Studio 17.10 installed and don't see "Ask Copilot" on a failing unit test, could you go that page and "Report a Problem" with a screenshot so that we can investigate?

![image](https://github.com/community/community/assets/10951869/d59869bc-6ce0-4dd8-937b-8e467f312853)


![image](https://github.com/community/community/assets/10951869/e3822ac3-b2d9-4d2f-8dcf-5fe2c27c65fd)

**Reply by Trinity-Kerr** [reply]
Okay cool, I will keep my eyes open for that feature to be released! I was able to get "explain test failure with Copilot" working.  Thank you!

**Reply by ayim** [reply]
Amazing, cheers!
